tests/pcap/Cargo.toml

```diff
@@ -6,14 +6,21 @@ publish = false
 
 [features]
 default = []
-ja4 = [] # Older versions of tshark do not support JA4
+# tshark >= 3.7.0 is required for JA3 support
```

Comment:
> I tested manually on my ubuntu18 instance

Is it worth the forking/complexity to support older platforms? Especially for a test only crate? We'll have to upgrade out instances as _some_ point, so this just kinda feels like putting off the inevitable.

Reply:
Yeah, I called that out in my description:
> Alternatively, we could just decide that the pcap tests don't support older wiresharks / older environments like Ubuntu18 and not feature gate anything. The build could just fail if tshark is too old.

The downside is that folks on the team who still use Ubuntu18 for testing will have issues if they ever want to run this test. I'm not sure how common that is though.

Reply:
I don't think it's a problem to merge this, but my personal vote is to axe the Ubuntu 18 support 🪓  and just require a newer tshark.

Reply:
Yeah, I think I agree. It's not like this will be the only test with specific runtime environment requirements.